index.js

```diff
@@ -136,8 +136,11 @@ function plugin (fastify, options, done) {
   done()
 }
 
-module.exports = fp(plugin, {
+module.exports = fp(fastifyBearerAuth, {
   fastify: '4.x',
   name: '@fastify/bearer-auth'
 })
+module.exports.default = fastifyBearerAuth
+module.exports.fastifyBearerAuth = fastifyBearerAuth
+
```

Comment:
`internals` what?!

Reply:
I think you mean the next line? Tbh. out of scope of this PR.

But maybe @climba03003 has an idea about it.

Reply:
We can focus on refactoring first. Then, update the types afterward.

Reply:
I though `internals` exist for test only. So, it calls `internals`.
https://github.com/fastify/fastify-bearer-auth/commit/a72107b859b7b448cd9fc784946f06892e7e5bab